support-frontend/assets/helpers/redux/checkout/product/selectors/isSupporterPlus.ts

```diff
@@ -26,6 +27,33 @@ export function isSupporterPlusPurchase(state: ContributionsState): boolean {
 	return amountIsHighEnough;
 }
 
+function isInHideBenefitsTest(abParticipations: Participations): boolean {
+	const regionalSuffixList: string[] = [
+		'UK',
+		'US',
+		'CA',
+		'EU',
+		'AU',
+		'NZ',
+		'ROW',
+	];
+
+	const result = regionalSuffixList.reduce((hideBenefits, suffix): boolean => {
```

Comment:
Yep can see how hideBenefits gradually reduces through the list of regional suffix's, nice!